From e426bffd2402b177d427810328ebbebceafe2ae3 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lothar=20Wa=C3=9Fmann?= <LW@KARO-electronics.de>
Date: Tue, 26 Mar 2019 09:36:22 +0100
Subject: [PATCH 4/8] regulator: bd71837: prevent warning when compiled with
 CONFIG_REGULATOR_DEBUG

The driver will be compiled with '-DDEBUG' when CONFIG_REGULATOR_DEBUG
is defined. This will lead to the following warning:
|drivers/regulator/bd71837-regulator.c:14:0: warning: "DEBUG" redefined
| #define DEBUG
|
|<command-line>:0:0: note: this is the location of the previous definition

Remove the unconditional '#define DEBUG' in the driver to prevent this
warning and utilize the standard means of enabling debugging for the
driver.
---
 drivers/regulator/bd71837-regulator.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/regulator/bd71837-regulator.c b/drivers/regulator/bd71837-regulator.c
index 0052f0212a27..7b94d1d73774 100644
--- a/drivers/regulator/bd71837-regulator.c
+++ b/drivers/regulator/bd71837-regulator.c
@@ -11,7 +11,6 @@
  *  option) any later version.
  *
  */
-#define DEBUG
 #include <linux/kernel.h>
 #include <linux/module.h>
 #include <linux/init.h>
-- 
2.20.1

